modules/xfce/hm.nix

```
@@ -0,0 +1,18 @@
+{ pkgs, config, lib, ... }@args:
+
+{
+  options.stylix.targets.xfce.enable =
+    config.lib.stylix.mkEnableTarget "Xfce" true;
```

The module needs to be disabled on Darwin to avoid errors.

```suggestion
    config.lib.stylix.mkEnableTarget "Xfce" pkgs.stdenv.hostPlatform.isLinux;
```